gpt_engineer/applications/cli/file_selector.py

```diff
@@ -221,10 +316,22 @@ def get_files_from_toml(self, input_path, toml_file):
         return selected_files
 
     def merge_file_lists(
-        self, existing_files: list[str], new_files: list[str]
```

Comment:
Out of curiosity, did you do this @kristiankyvik , or was this an automatic type correction?

Reply:
This was GPT-engineer. Despite me explicitly requesting it to limit itself to adding/editing docstrings, in some cases, it took the liberty of also correcting the parameter types. I figured I would also keep these suggestions in the PR after manually reviewing them.